src/cmd/linuxkit/push_azure.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+// Process the run arguments and execute run
+func pushAzure(args []string) {
+	flags := flag.NewFlagSet("azure", flag.ExitOnError)
+	invoked := filepath.Base(os.Args[0])
+	flags.Usage = func() {
+		fmt.Printf("USAGE: %s push azure [options] [name]\n\n", invoked)
+		fmt.Printf("'name' specifies the path (absolute or relative) of a\n")
+		fmt.Printf("VHD image be uploaded to an existing Azure Storage Account\n")
+		fmt.Printf("Options:\n\n")
+		flags.PrintDefaults()
+	}
+
+	resourceGroupName := flags.String("resourceGroupName", "", "Name of resource group to be used for VM")
+	accountName := flags.String("accountName", "linuxkitstorage", "Name of the storage account")
```

Comment:
is this going to be created if it doesn't exist. Not sure if it's a good idea to hard code the default of `linuxkitstorage`